subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -1059,6 +1060,11 @@ static inline struct ll_sync_iso_set *
 
 static void done_disabled_cb(void *param)
 {
+	struct ll_scan_aux_set *aux;
+
+	aux = param;
+	LL_ASSERT(aux->parent);
+
 	flush(param);
```

Comment:
Would it make sense to do `flush(aux)` now that we have a more specific name? (not needed at all). 

If not, I guess we could also add `const` to `struct ll_scan_aux_set *aux;` or even shorten this to 
`LL_ASSERT(((struct ll_scan_aux_set *)param)->parent)`